lab-09-1-xor.py

```diff
@@ -3,33 +3,22 @@
 import numpy as np
 
 tf.set_random_seed(777)  # for reproducibility
-learning_rate = 0.1
 
-x_data = [[0, 0],
-          [0, 1],
-          [1, 0],
-          [1, 1]]
-y_data = [[0],
-          [1],
-          [1],
-          [0]]
-x_data = np.array(x_data, dtype=np.float32)
-y_data = np.array(y_data, dtype=np.float32)
+x_data = np.array([[0, 0], [0, 1], [1, 0], [1, 1]], dtype=np.float32)
+y_data = np.array([[0], [1], [1], [0]], dtype=np.float32)
 
-X = tf.placeholder(tf.float32, [None, 2])
-Y = tf.placeholder(tf.float32, [None, 1])
+X = tf.placeholder(tf.float32, [4, 2])
```

Comment:
It was intentionally set to `None` since the number of records is usually not fixed.

```suggestion
X = tf.placeholder(tf.float32, [None, 2])
```